dogfooding/lib/screens/login_screen.dart

```diff
@@ -268,3 +279,12 @@ String randomId({int size = 21}) {
   }
   return id;
 }
+
+String createValidId(String id) {
+  return id
+      .replaceAll('@', '_')
+      .replaceAll('.', '_')
+      .replaceAll('.', '_')
```

Comment:
```suggestion
      .replaceAll('.', '_')
```

Is this repeated?

Reply:
@xsahil03x it could be at least like: `john.black@mail.com` - 2 dots

Reply:
But I assume the second `.replaceAll('.', '_')` is redundant :)

Reply:
replaced it with something more elegant ;)